signer/package.json

```diff
@@ -4,6 +4,9 @@
   "description": "",
   "author": "SingularityNet",
   "dependencies": {
-  	"singularitynet-platform-contracts": "0.3.4"
+  	"serverless-offline": "5.12.0",
```

Comment:
i ahve removed it @prashantramangupta  please check if this is still reuqired

Reply:
I need this dependency "singularitynet-platform-contracts"